content/millennium/r4/foundation/other/health-cards.md

```diff
@@ -43,11 +43,11 @@ Represents the patient and the clinical data related to a vaccination
 
 Contains the following resources:
 
-* A single Patient resource that follows the [VaccinationCredentialPatientDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-dm.html#tab-ms) profile with the following fields, if valued:
+* A single Patient resource that follows the [VaccinationCredentialPatientDM](https://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-patient-us-resource-examples.html) profile with the following fields, if valued:
   * [Patient official name](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.name){:target="_blank"}
   * [Date of Birth](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.birthDate){:target="_blank"}
 
-* One or more Immunization resources that follow the [VaccinationCredentialImmunizationDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-immunization-dm.html) profile with the following fields, if valued:
+* One or more Immunization resources that follow the [VaccinationCredentialImmunizationDM](https://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-vaccination-dm.html) profile with the following fields, if valued:
```

Comment:
Just checking, is this link correct here? I'm not sure whether Alec pasted the same link for this and VaccinationCredentialPatientDM by mistake and if they should be different or not.

Reply:
> Just checking, is this link correct here? I'm not sure whether in the Jira comment Alec pasted the same link for this and VaccinationCredentialPatientDM by mistake and if they should be different or not.

Hi Florian, the new links are not the same : 
- For VaccinationCredentialImmunizationDM : https://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-vaccination-dm.html
- For VaccinationCredentialPatientDM            : https://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-patient-us-resource-examples.html

Reply:
> > Just checking, is this link correct here? I'm not sure whether in the Jira comment Alec pasted the same link for this and VaccinationCredentialPatientDM by mistake and if they should be different or not.
> 
> Hi Florian, the new links are not the same :
> 
>     * For VaccinationCredentialImmunizationDM : [build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-vaccination-dm.html](https://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-vaccination-dm.html)
> 
>     * For VaccinationCredentialPatientDM            : [build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-patient-us-resource-examples.html](https://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-patient-us-resource-examples.html)

I think you should probably not use the 'build' version as the links will keep breaking. I think we should probably use the links from 2021Sep guide http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-vaccination-dm.html and http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-patient-us-dm.html